tests/test_dataplex_clouddq_task_integration.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from clouddq.integration.dq_dataplex import DqDataplex
+import json
+import time
+import datetime
+
+import logging
+
+# Create and configure logger
+logging.basicConfig(format='%(asctime)s %(message)s')
+
+# Creating an object
+logger = logging.getLogger()
+
+# Setting the threshold of logger to DEBUG
+logger.setLevel(logging.DEBUG)
+
+task_id = f"test-task-1-{datetime.datetime.utcnow()}".replace(" ", "utc").replace(":", "-").replace(".", "-")
+
+class TestDataplexIntegration:
+
+    @pytest.fixture
+    def test_dq_dataplex(self):
+
+        dataplex_endpoint = "https://dataplex.googleapis.com"
```

Comment:
move endpoint into default of class. make optional variable to override endpoint.